src/open_inwoner/templates/pages/contactmoment/detail.html

```diff
@@ -1,21 +1,44 @@
 {% extends 'master.html' %}
-{% load i18n anchor_menu_tags card_tags file_tags grid_tags table_tags solo_tags form_tags button_tags %}
+{% load i18n grid_tags table_tags button_tags icon_tags %}
 
 {% block content %}
     {% if contactmoment %}
         {% render_grid %}
             {% render_column span=12 %}
-                {# Contactmoment/dashboard. #}
-                <h1 class="h1" id="title">{{ page_title }}</h1>
+                <h1 class="h1" id="title">{{ contactmoment.onderwerp }}</h1>
                 {% include "components/Dashboard/Dashboard.html" with metrics=metrics only %}
             {% endrender_column %}
-            {% render_column start=4 span=6 %}
+        {% endrender_grid %}
 
-                <div class="contactmoment__details">
-                {% contactmoment_table contactmoment zaak=zaak %}
+        {% render_grid %}
+            {% render_column start=4 span=6 %}
+                <div class="contactmoment">
+                    <h2 class="contactmoment__heading">{% trans "Uw vraag" %}</h2>
+                    <p class="contactmoment__text">{{ contactmoment.text }}</p>
                 </div>
+                <div class="contactmoment">
+                    <h2 class="contactmoment__heading">{% trans "Antwoord" %}</h2>
+                    {% if contactmoment.antwoord %}
+                        <p class="contactmoment__text">{{ contactmoment.antwoord }}</p>
+                    {% else %}
+                        <p class="contactmoment__text">{% trans "Nog geen antwoord beschijkbaar" %}</p>
+                    {% endif %}
+                </div>
+                <div class="contactmoment">
+                    <a href="{{ origin.url }}" id="origin_link" class="button button--secondary">
+                        {% icon "arrow_backward" %}{{ origin.label }}
+                    </a>
+                </div>
+                {% if destination %}
+                    <div class="contactmoment">
+                        <a href="{{ destination.url }}" id="destination_link" class="button button--transparent">
```

Comment:
Both your button--transparent, as well as your primary button need to be Bold - This style looks a lot like what we already use for Forms with a fullwidth actions area. You should wrap these two buttons with a Div with class `contactmoment__actions` so you can give them both the bold font-weight.